src/main/java/org/spongepowered/common/mixin/core/item/MixinItemFishingRod.java

```diff
@@ -54,10 +54,12 @@ public ItemStack onItemRightClick(ItemStack itemStack, World world, EntityPlayer
             player.fishEntity.handleHookRetraction();
         } else {
             EntityFishHook fishHook = new EntityFishHook(world, player);
-            if (!Sponge.getGame().getEventManager()
+            if (world.isRemote || !Sponge.getGame().getEventManager()
```

Comment:
@Zidane 
> The lack of specifying the property in the `@Subscribe` would equate to it being whatever would normally happen otherwise.

However, a value would have to be specified in either the `Subscribe.java` or the method using `@Subscribe`, and as far as I know, there's no way to differentiate between them.

> There are some events that will never fire on one side so introducing BOTH is equally un-necessary.

But aren't there some events that fire on both sides?